src/main/java/org/springframework/data/couchbase/core/ExecutableExistsByIdOperationSupport.java

```diff
@@ -32,17 +32,25 @@
 
 	@Override
 	public ExecutableExistsById existsById() {
-		return new ExecutableExistsByIdSupport(template, null);
+		return new ExecutableExistsByIdSupport(template, null, null, null);
 	}
 
 	static class ExecutableExistsByIdSupport implements ExecutableExistsById {
 
 		private final CouchbaseTemplate template;
+		private final String scope;
+		private final String collection;
+		private final GetOptions options;
```

Comment:
ExistsOptions?

Reply:
Yes.  Similar issue with RemoveByQuery - it had RemoveOptions instead of QueryOptions - and in both cases the options were not passed in execution.   Going through all the  Reactive*OperationSupport classes to check for usage of options.

Reply:
I added "options" tests that pass an options with timeout of 10 nanoseconds, resulting in a timeout.